images/php-fpm/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:7.2-fpm
+FROM php:7.2.6-fpm-stretch
```

Comment:
I'd like the patch version to auto update itself so `7.2-fpm` was fine. Also; `7.2-fpm === 7.2-fpm-stetch`

Reply:
Specific versioning is important when performing this level of custom compiling. 7.2-fpm was recently 7.2-fpm-jessie (Debian Jessie). Within the past few weeks, it switched to 7.2-fpm-stretch (Debian Stretch). There's also a few other distros in that image repo (e.g. Alpine). There is no guarantee the 7.2 tag won't switch distros at any given moment. I *highly* recommend at least being specific with the distro we're targeting when we're making this level of customization to an image build.

Reply:
https://github.com/vanilla/vanilla-docker/commit/acdd64535d5054334a9afa7c588a929e5fb7516e